site/_posts/2021-04-01-why-user-multi-factor-authentication.md

```diff
@@ -0,0 +1,129 @@
+
+However, the focus of this post is MFA in the context of online account access. Because of that and the prevalence of passwords, this post will assume that your users' first factor is a password. 
+
+Let's talk more about why you, as a developer, should enable and encourage your users to use MFA.
+
+## Why use MFA?
+
+A foundational part of building a secure, available system is ensuring only authorized people and software agents access it. Both authentication, which ensures that users are who they say they are, and authorization, which controls access, are critical to achieving this goal. Consider what a compromised account could do to your systems and your users' data, then think about how MFA can help prevent that.
+
+If your users provide only one authentication factor, it can be stolen by a bad actor. This actor now has the same access to the system as the user does.
+
+Secrets users know, such as passwords, are being stolen regularly. While systems can help prevent unauthorized access by [detecting stolen passwords](/learn/expert-advice/security/breached-password-detection/) and users may protect themselves by practicing good password hygiene, requiring another factor increases the security of user accounts on your system. Microsoft researchers found that accounts are ["99.9% less likely to be compromised"](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/your-pa-word-doesn-t-matter/ba-p/731984) if MFA is used.
+
+Implementing MFA is typically a partnership with your users. Some forms of MFA are easier for system developers to implement. Others require more effort from users.
+
+### The balance between user experience and account security
+
+Even though MFA is more secure, as a developer, be cautious about requiring it for all system access. In this situation, you are engaged in a fundamental engineering practice: making a tradeoff. 
+
+You want the user login experience to be as smooth as possible while minimizing account takeover opportunities. Friction in the authentication process will annoy some percentage of your users. They are using your application to do a job, not because of deep love of your secure login process. 
+
+Listen to your users when you are discussing MFA requirements. You don't want them to circumvent MFA. At the same time they may need to be educated on the benefits. How many of you know people who still write down passwords on post-it notes? I do.
+
+Acknowledge and address the balance between the friction of the user login experience and the risk of account compromise. Do so in the context of the data your system secures; different data requires different levels of assurance. If your site lets users vote on cat pictures, MFA may never be required. If your site lets users transfer money, or digital assets similar to money, to complete strangers on the internet, on the other hand, it should require MFA.
```

Comment:
If your site lets users transfer money, or digital assets similar to money, to complete strangers on the internet, on the other hand, it should require MFA. 

Consider revising this sentence?  Lots of commas